### PR TITLE
GSTIN and OrderInstructions support

### PR DESCRIPTION
**Pull request by angbpy**

### CODE REVIEW

File: woo-razorpay.php [hunk 1]
```diff
@@ -1677,8 +1677,15 @@ public function update1ccOrderWC(& $order, $wcOrderId, $razorpayPaymentId)
             $razorpayData = $api->order->fetch($razorpayOrderId);
 
             $this->UpdateOrderAddress($razorpayData, $order);
+            
+            $razorpayData['notes']['gstin']             = "MOCKGST-32626092022";
+            $razorpayData['notes']['order_instruction'] = "MOCKOrderInstructions";
 
-
+            $gstin             = $razorpayData['notes']['gstin'];
```
Comment (poojarazor) [inline woo-razorpay.php:1684]: Please empty field condition also

File: woo-razorpay.php [hunk 2]
```diff
@@ -1677,8 +1677,15 @@ public function update1ccOrderWC(& $order, $wcOrderId, $razorpayPaymentId)
...
+            $order_instruction = $razorpayData['notes']['order_instruction'];
```
Comment (poojarazor) [inline woo-razorpay.php:1685]: formatting $orderInstruction

File: woo-razorpay.php (hunk 1 above)
Comment (poojarazor) [inline woo-razorpay.php:1684]: make it gstNo

File: woo-razorpay.php [hunk 3]
```diff
@@ -1677,8 +1677,15 @@ public function update1ccOrderWC(& $order, $wcOrderId, $razorpayPaymentId)
             $razorpayData = $api->order->fetch($razorpayOrderId);
 
             $this->UpdateOrderAddress($razorpayData, $order);
+            
+            $razorpayData['notes']['gstin']             = "MOCKGST-32626092022";
+            $razorpayData['notes']['order_instruction'] = "MOCKOrderInstructions";
```
Comment (poojarazor) [inline woo-razorpay.php:1682]: remove dummy

Comment (angbpy) [reply woo-razorpay.php]: Removed

File: woo-razorpay.php (hunk 1 above)
Comment (angbpy) [reply woo-razorpay.php]: Added null check

Comment (angbpy) [reply woo-razorpay.php]: Done

**Review by poojarazor — APPROVED** [review]
LGTM

**Pull request merged**